libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/EHorizonMapper.kt

```diff
@@ -65,9 +65,9 @@ internal suspend fun RoadObjectDistanceInfo.mapToEHorizonObjectDistanceInfo():
  */
 internal fun RoadObjectLocation.mapToEHorizonObjectLocation(): EHorizonObjectLocation {
     return EHorizonObjectLocation(
-        edges,
-        percentAlongBegin,
-        percentAlongEnd
+        path?.edges ?: emptyList(),
+        position?.percentAlong ?: 0.0,
+        position?.percentAlong ?: 0.0
```

Comment:
@korshaknn flagging that there are some EH-related breaking changes. I've hacked this to get my branch working but this needs to be properly addressed when the new NN version is available.

Same as above, what NN version are `'da-common-runloop-callback-3-SNAPSHOT'` and `'da-common-scheduler-callback-5-SNAPSHOT'` artifacts based off? Latest `master`? Could you clarify @DmitryAk?

Reply:
The same, the artifacts are slightly obsolete (from the 2nd March) - I'll refresh them today.